snakemake/__init__.py

```diff
@@ -2697,6 +2702,18 @@ def adjust_path(f):
         print("", file=sys.stderr)
         sys.exit(1)
 
+    if args.force_local:
+        args.cluster = None
+        args.slurm = False
+        args.slurm_jobstep = False
+        args.cluster_sync = None
+        args.tibanna = False
+        args.kubernetes = None
+        args.tes = None
+        args.google_lifesciences = False
+        args.drmaa = None
+        args.flux = False
```

Comment:
This is dangerous, as we might miss a future backend here. A safer way would be to pass force_local to the scheduler and let it override all other cluster/cloud statements considered there.

Apart from that: what about just having a second profile for the local execution? Wouldn't that be sufficient?

Reply:
Hi @johanneskoester, I agree this might be missed in the future. So I just made a commit that integrated it with the run_local variable that seems to be used in `Workflow` that should accomplish the same thing. I am not super familiar with the codebase so I apologize if this is not what you were suggesting. 

I agree a second profile is a pretty reasonable option to get around this, but on the other hand if this solution could be made acceptable I would much prefer a command line argument. 

Thanks!
Mitchell